### ISSUE TITLE
About loss design

### ISSUE THREAD

**Issue by llan-ml** [tkipf/gcn#120]
Hi @tkipf . I saw a comment that in practice we can use the representations learned at each layer to construct a compound loss function, e.g., the summation of three cross-entropy loss function if we have three gc layers.

I don't remember where I saw this comment, and it seems that this way can make the convergence more stable when we have no node features. Could you give some opinions on this? Thanks.

**Comment by tkipf** [comment]
I do not remember seeing such a suggestion. Not sure exactly if I
understand what you mean, but if you want such an approach to work you
would probably have to attach a softmax classification layer + cross
entropy loss after each graph propagation layer.
